demisto_sdk/commands/validate/validators/IF_validators/IF111_is_field_type_changed.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from typing import Iterable, List, cast
+
+from demisto_sdk.commands.common.constants import GitStatuses
+from demisto_sdk.commands.content_graph.objects.incident_field import IncidentField
+from demisto_sdk.commands.validate.validators.base_validator import (
+    BaseValidator,
+    FixResult,
+    ValidationResult,
+)
+
+ContentTypes = IncidentField
+
+
+class IsFieldTypeChangedValidator(BaseValidator[ContentTypes]):
+    error_code = "IF111"
+    description = "Checks if the field type changed"
+    rationale = "Changing type of IncidentField is not allowed by the platform"
+    error_message = "Changing incident field type is not allowed"
+    fix_message = "The `type` field has changed back"
```

Comment:
```suggestion
    fix_message = "Changed the `type` field back to {0}."
```
Where 0 is the old type.